### PR TITLE
Added Profile component

### PR DESCRIPTION
**Pull request by redmyt**
## JIRA

* [Main JIRA ticket](https://ssu-jira.softserveinc.com/browse/LVPYTP-57)

## Code reviewers

- [ ] @dmytrorevak
- [ ] @IvanAndrukh
- [ ] @JjuliaP
- [ ] @makarman
- [x] @noobhunters
- [ ] @Silent777
- [ ] @Staryj812

### Second Level Review

- [ ] @ssOleg
- [ ] @lhalam

## Summary of issue

Created Profile container component
Created ItemsList subcomponent
Created ItemsListTitle subcomponent
Created ItemsUnit subcomponent
Created MessageItem subcomponent
Created MessagesList subcomponent
Created MessagesBar component
Created profileService file
Created basic ProgressGraph component
Created two temporary components MentrorsList and MentorItem

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: eventually/static/src/containers/profile/MessagesBar.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        }
+    }
+
+    componentWillMount() {
+        this.setState({
+            mentors: mentorsService().mentors
+        })
+    }
+
+    handleMentorClick = (mentorId) => {
```
Comment (lhalam) [inline eventually/static/src/containers/profile/MessagesBar.js]: if one parameter does not need to be used `()`
`handleMentorClick = mentorId => {`

File: eventually/static/src/containers/profile/ItemUnit.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+export default class ItemUnit extends React.Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            expanded: false,
+        };
+    }
+
+    handleExpandChange = (expanded) => {
```
Comment (lhalam) [inline eventually/static/src/containers/profile/ItemUnit.js]: if one parameter does not need to be used `()`
`handleMentorClick = mentorId => {`

**Review by lhalam — DISMISSED** [review]

File: eventually/static/src/containers/profile/ItemUnit.js (hunk 2 above)
Comment (redmyt) [reply eventually/static/src/containers/profile/ItemUnit.js]: fixed

File: eventually/static/src/containers/profile/MessagesBar.js (hunk 1 above)
Comment (redmyt) [reply eventually/static/src/containers/profile/MessagesBar.js]: done

**Review by skramar — APPROVED** [review]

**Review by lhalam — APPROVED** [review]

**Pull request merged**